tests/pages/settings/profile-page.ts

```diff
@@ -1,12 +1,19 @@
-import { create, triggerable, visitable } from 'ember-cli-page-object';
+import { clickable, create, triggerable, visitable } from 'ember-cli-page-object';
+import AccountDropdown from 'codecrafters-frontend/tests/pages/components/account-dropdown';
 
 export default create({
+  accountDropdown: AccountDropdown,
+
   profileDescription: {
     input: {
       blur: triggerable('blur'),
       scope: '[data-test-profile-description-input]',
     },
   },
 
+  anonymousModeToggle: {
```

Comment:
Alphabetical ordering - place this below `accountDropdown`, above `profileDescription`